samples/sdl_ttf/main.c

```diff
@@ -0,0 +1,241 @@
+/*
+  Copyright (C) 1997-2016 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely.
+*/
+/* Simple program:  Create a native window and attach an SDL renderer */
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <hal/video.h>
+#include "stdio.h"
+#include "string.h"
+#include <SDL.h>
+#include <SDL_ttf.h>
+
+int _exit(int x)
+{
+    debugPrint("_exit called\n");
+    debugPrint( " SDL_Error: %s\n", SDL_GetError() );
+    XSleep(5000);
+    XReboot();
+    return 0;
+}
+
+//Screen dimension constants
+const extern int SCREEN_WIDTH;
+const extern int SCREEN_HEIGHT;
+#define NUM_SPRITES 10
+#define MAX_SPEED   5
+
+static SDL_Rect *positions, *velocities;
+
+/* Call this instead of exit(), so we can clean up SDL: atexit() is evil. */
+static void
+quit(int rc)
```

Comment:
This `rc` parameter remains unused.

Whenever this or `_exit` are being called, there are random numbers in the code - which is confusing to new users.